polyhost/input/win_helper.py

```
@@ -17,50 +21,84 @@ def __init__(self, poly_settings=None):
-                    if e.startswith('LanguageTag'):
-                        self.list.append(e.split(":")[-1].strip())
+                # Emit an explicit "LanguageTag: <tag>" line per language rather
+                # than letting PowerShell default-format the list — with 2+
+                # languages it renders as a TABLE whose header row also starts
+                # with "LanguageTag" (and carries no value), the same formatting
+                # trap that broke get_current_language. ForEach-Object is
+                # formatting- and locale-independent.
+                result = subprocess.run(
+                    ['powershell', '-NoProfile', '-NonInteractive', '-WindowStyle', 'Hidden',
+                     '-Command',
+                     "Get-WinUserLanguageList | ForEach-Object { 'LanguageTag: ' + $_.LanguageTag }"],
```

Ruff (0.15.17)

[error] Starting a process with a partial executable path

(S607)